libunwind/test/floatregister.pass.cpp

```diff
@@ -11,20 +11,27 @@
 
 // Basic test for float registers number are accepted.
 
-#include <dlfcn.h>
 #include <stdlib.h>
 #include <string.h>
 #include <unwind.h>
 
+// Using __attribute__((section("main_func"))) is Linux specific, but then
```

Comment:
Nit: This is not specific to Linux it is specific to the behaviour of ELF linkers that automatically create these symbols for sections that are valid C identifiers.

Reply:
Fair point.